phalcon/messages/messages.zep

```diff
@@ -201,7 +201,7 @@ class Messages implements \Countable, \ArrayAccess, \Iterator, \JsonSerializable
 	 *
 	 * @param int index
 	 */
-	public function offsetExists(string index) -> boolean
+	public function offsetExists(var index) -> boolean
```

Comment:
Previously you removed type hinting for all `offset*` methods in the `Phalcon\Di` class.

Reply:
this specific change is strange, this code is being removed with https://github.com/phalcon/cphalcon/pull/13534 
And it doesn't exists in my code. I assume cuz I merged 3.4.x changes in here, after merge this should be gone. No idea what will happen when merged xD

Reply:
@Izopi4a Well, could you rebase you branch to make sure that there is no collision?

Reply:
I can't make it work ( i have never used rebase actually ). well I am off for a week when I come back I will create a new branch and do the work from scratch. Because I am not sure what will happen if this gets merged